src/assets/homePageImg/Icons/BagIcon.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If the file is inside the "icon" folder, there's no need for the filename to include "icon".